data/data.qrc

```diff
@@ -350,6 +350,7 @@
         <file>providers/22x22/skydrive.png</file>
         <file>providers/22x22/somafm.png</file>
         <file>providers/22x22/intergalacticfm.png</file>
+        <file>providers/22x22/radiobrowser.png</file>
```

Comment:
Do you know what license these images carry?

Reply:
I took it from [here](https://github.com/segler-alex/radiobrowser-api-rust/blob/master/static/favicon.ico), and the project is using GNU Affero.
Other projects from the same author released under GPL v3 contains the same icon, eg: https://github.com/segler-alex/WeRadio/tree/master/radio

Reply:
Should be good, then.